weblate/static/style-bootstrap.css

```diff
@@ -1978,3 +1978,7 @@ label .text-muted {
   transition: all 300ms linear;
   -webkit-transition: all 300ms linear;
 }
+
+#add-glossary-form .highlighted-output {
+  font-size: 16px;
+}
```

Comment:
This was an unintentional CSS effect, I've fixed that in 926058cd09, so this can be removed now.

```suggestion
```